job/handlers/resources/WindowsServer_2016/gitRepo/templates/providers/_github.ps1

```diff
@@ -118,11 +118,9 @@ Function git_sync() {
 
   popd
 
-  echo "----> Copying to $PROJECT_CLONE_LOCATION"
-  Copy-Item "$temp_clone_path\*" -Destination $PROJECT_CLONE_LOCATION -Recurse
+  echo "----> Moving to $PROJECT_CLONE_LOCATION"
+  Move-Item "$temp_clone_path" "$PROJECT_CLONE_LOCATION"
```

Comment:
`Copy-Item` wasn't copying because of the wildcard match?

Reply:
It needs `-Force` to copy hidden and system files. I didn't see the point in preserving the copy. We're removing the source location immediately anyway, so move seemed like a logical choice.